obspy/core/inventory/response.py

```diff
@@ -961,6 +961,51 @@ def is_close(a, b):
 
         return sampling_rates
 
+    @classmethod
+    def get_flat_response(cls, sensitivity=1., frequency=1.,
+                          input_units="M/S", output_units="M/S",
+                          include_pole_zero_stage=False):
+
+        """
+        Makes a flat response object.
+
+        :type sensitivity: float, optional
+        :param sensitivity : The sensitivity of the response, defaults to 1.
+        :type frequency: float, optional
+        :param frequency: The frequency where the sensitivity is stated,
+            defaults to 1 Hz.
+        :type input_units: string, optional
+        :param input_units: The input units going into the response object,
+            default units are "M/S"
+        :type output_units: string, optional
+        :param input_units: The output units going out of the response object,
+            default units are "M/S"
+        :type include_pole_zero_stage: bool, optional
+        :param include_poles_zero_stage: This option allows for a pole/zero
+            stage to be include. If a pole zero stage is included it adds
+            one pole and one zero both with the value of 0 + 1j.  This allows
+            for the user to apply a deconvolution routine.  E.g. it allows
+            the user to include the tapering, detrendings, and filtering
+            during the response removal.
+        """
+        sensitivity = InstrumentSensitivity(sensitivity, frequency,
+                                            input_units, output_units)
+        stage = []
+        if include_pole_zero_stage:
+            pzt = "laplace hz"
+            stg1 = PolesZerosResponseStage(stage_sequence_number=1,
+                                           stage_gain=sensitivity,
```

Comment:
stage gain is just a plain float.

```
import copy
from obspy import read, read_inventory
from obspy.core.inventory import Response
st = read()
inv = read_inventory()
inv = inv.select(station='RJOB', time=st[0].stats.starttime)
flat = Response.get_flat_response(include_pole_zero_stage=True)
for cha in inv[0][0]:
    cha.response = copy.deepcopy(flat)
inv.plot_response(0.001)

Traceback (most recent call last):
  File "flat.py", line 10, in <module>
    inv.plot_response(0.001)
  File "/home/megies/git/obspy/obspy/core/inventory/inventory.py", line 1100, in plot_response
    outfile=None)
  File "/home/megies/git/obspy/obspy/core/inventory/channel.py", line 366, in plot
    outfile=outfile)
  File "/home/megies/git/obspy/obspy/core/inventory/response.py", line 1841, in plot
    end_stage=end_stage)
  File "/home/megies/git/obspy/obspy/core/inventory/response.py", line 1703, in get_evalresp_response
    freqs, output=output, start_stage=start_stage, end_stage=end_stage)
  File "/home/megies/git/obspy/obspy/core/inventory/response.py", line 1665, in get_evalresp_response_for_frequencies
    end_stage=end_stage)
  File "/home/megies/git/obspy/obspy/core/inventory/response.py", line 1554, in _call_eval_resp_for_frequencies
    gain_blkt.gain = blockette.stage_gain
TypeError: a float is required
```

Reply:
Thanks, I fixed this.  I was writing over my sensitivity unknowingly.